Source/ASTextNode.mm

```diff
@@ -1233,12 +1234,17 @@ - (void)_invalidateTruncationText
 {
   {
     ASLockScopeSelf();
-    _composedTruncationText = nil;
+    [self _locked_invalidateTruncationText];
   }
 
   [self setNeedsDisplay];
```

Comment:
I feel like now is the time to let callers of this (unlocked) method to call `-setNeedsDisplay` themselves. It's a bit confusing to only automatically do it with the unlock version.